lib/usbmux/index.js

```diff
@@ -61,14 +61,26 @@ function swap16 (val) {
  * @throws {B.TimeoutError} if connection timeout happened
  * @returns {Promise<NodeJS.Socket>} Connected socket instance
  */
-async function getDefaultSocket (opts = {}) {
-  const {
+async function getDefaultSocket(opts = {}) {
+  let {
     socketPath = DEFAULT_USBMUXD_SOCKET,
     socketPort = DEFAULT_USBMUXD_PORT,
     socketHost = DEFAULT_USBMUXD_HOST,
     timeout = 5000,
   } = opts;
 
+  if (process.env.USBMUXD_SOCKET_ADDRESS) {
```

Comment:
I think it would be useful to add a debug log line here about loading the value from env var